src/validation.h

```diff
@@ -300,17 +300,26 @@ bool CheckFinalTxAtTip(const CBlockIndex* active_chain_tip, const CTransaction&
  *                              mempool and chainstate UTXO set. In the latter case, the caller is
  *                              responsible for holding the appropriate locks to ensure that
  *                              calls to GetCoin() return correct coins.
+ *
+ * @returns The resulting height and time calculated and the hash of the block needed for calculation.
+ */
+std::optional<LockPoints> CalculateLockPointsAtTip(CBlockIndex* tip,
+                                                   const CCoinsView& coins_view,
+                                                   const CTransaction& tx);
+
+/**
+ * Check if transaction will be BIP68 final in the next block to be created on top of tip.
+ * @param[in]   tip             Chain tip to check tx sequence locks against. For example,
+ *                              the tip of the current active chain.
+ * @param[in]   lp              Relative locktime constraints (BIP68).
```

Comment:
in 5a6ce8cca31b3105cb8e96cd3e5e6c0da0fb3479

```suggestion
 * @param[in]   lp              LockPoints containing the height and time at which this transaction is final.
```

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/23897#issuecomment-1191310762).